thorlcr/activities/group/thgroupslave.cpp

```diff
@@ -61,6 +61,8 @@ class GroupSlaveActivity : public CSlaveActivity
         numGroups = 0;
         numGroupMax = 0;
         startLastGroup = 0;
+        if (container.queryLocalOrGrouped())
```

Comment:
Yes, global case requires a mptag, that's used to communicate to the rollover of the end group to next slave.